modules/azurerm_postgresql_flex/variables.tf

```diff
@@ -98,9 +98,3 @@ variable "delegated_subnet_id" {
   description = "The ID of the virtual network subnet to create the PostgreSQL Flexible Server. The provided subnet should not have any other resource deployed in it and this subnet will be delegated to the PostgreSQL Flexible Server, if not already delegated. Changing this forces a new PostgreSQL Flexible Server to be created."
   type        = string
 }
-
-variable "private_dns_zone_id" {
-  description = "The ID of the private DNS zone to create the PostgreSQL Flexible Server. Changing this forces a new PostgreSQL Flexible Server to be created."
-  type        = string
-  default     = null
-}
```

Comment:
No longer needed? This is unclear.

Reply:
same as above.
Variable was specified in the `variables.tf` file but was not used in main.tf, hence tflint throws `terraform_unused_declarations` warning. Removing this does not affect the functionality.